extensions/amp-truncate-text/0.1/amp-truncate-text-shadow.css

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+:host {
+  padding: 0 !important;
+  border: none !important;
+}
+
+:host([i-amphtml-truncate-expanded]) {
```

Comment:
I don't think this is a thing at this point. I think they replaced it with `:host-context`, but even that's not yet fully agreed to and only Chromium supports.

Reply:
Sounds good. Let's do this and we'll research it more.